app/views/vendors/show.html.erb

```diff
@@ -36,42 +36,42 @@
 
   </div>
 
-  </div>
-
-
-
-
-
+</div>
 
   <div class="row">
 
   <div class="large-12 columns">
    <div class="radius panel">
 
+   <div>
+     <h3>Reviews:</h3>
+     <% @reviews.each do |review| %>
+       <div>
+         <%= review.rating %>
+         </br>
```

Comment:
Don't use `<br>` tags - bad form! Try taking it out, and if it doesn't look like you want it to, try wrapping line 50 and 52 each in their own (more descriptive) HTML tag instead of using `<br>`